services/nfs-server-provisioner/0.6.0/defaults/cm.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   namespace: ${releaseNamespace}
 data:
   values.yaml: |-
+    priorityClassName: "dkp-high-priority"
```

Comment:
looking at the chart [here](https://github.com/mesosphere/charts/tree/master/staging/nfs-server-provisioner), it doesn't look like it supports setting `priorityClassName` yet. created https://d2iq.atlassian.net/browse/D2IQ-97213 to add it